week1/scary/maverickchaser/Main.hs

```diff
@@ -0,0 +1,16 @@
+import System.IO
+import Data.Char
+
+magickNumber = 13
+fPath = "/usr/share/dict/words"
+
+isScary :: [Char] -> Bool
+isScary word
+    | isAlphaOnly word && foldl (\acc c -> acc + ord (toUpper c) - ord 'A' + 1) 0 word == magickNumber = True
```

Comment:
Also some of the common folds (ones with `(+)`, `(*)`, `(++)`, `(&&)`, `(||)`, `min`, `max`) have their own names: `sum`, `product`, `concat`, `and`, `or`, `minimum`, `maximum`. It's better to use an existing name than a raw fold (and a raw fold is preferable to recursion). So, in this case it would be 

``` haskell
sum (map (\c -> ord (toUpper c) - ord 'A' + 1) word)
```

(and then you'd want to move the inner lambda into a separate function in a `where` because it's nicer this way).